### PR TITLE
Avatar and userId are affiliated

### PR DESCRIPTION
**Pull request by Unikorn44**

### CODE REVIEW

File: src/Controller/AvatarController.php [hunk 1]
```diff
@@ -8,9 +8,11 @@ class AvatarController extends AbstractController
 {
     public function avatarCreate()
     {
-        // On crécupère le userId pour identification
-        $_SESSION ['userId'] = 3;
+        // On récupère le userId pour identification
+        //$_SESSION ['userId'] = 3;
         $string = $_SESSION ['userId'];
+    //var_dump($string);
```
Comment (aldaitzwild) [inline src/Controller/AvatarController.php:14]: No dead code !

File: src/View/Profiles/profile.html.twig [hunk 2]
```diff
@@ -12,7 +12,10 @@
 				<td align="center">
 					<br/> <!--random avatar creation button-->
 					<input type="button" value="Nouvel avatar"/ onclick="window.location.href = '/avatar/avatarCreate';">
-					<img src="../assets/images/avatars/avatar3.png"> <!--création image avatar3.png-->
+					<!--document.body.innerHTML = "<img src=\"img/" + item + ".jpg\" alt=\"" + item + ".jpg\" />";-->
+					<!--$road = 'assets/images/avatars/avatar' . $string . '.png';-->
+
```
Comment (aldaitzwild) [inline src/View/Profiles/profile.html.twig:17]: More dead code !

**Review by aldaitzwild — CHANGES_REQUESTED** [review]

**Comment by Unikorn44** [comment]
Dead code removed

**Comment by Unikorn44** [comment]
Checked

**Review by aldaitzwild — CHANGES_REQUESTED** [review]
You have conflict with Dev : 

Conflicting files
public/assets/images/avatars/avatar3.png
src/View/Profiles/profile.html.twig

**Comment by Unikorn44** [comment]
Conflicts resolved

File: src/Controller/AvatarController.php [hunk 3]
```diff
@@ -8,17 +8,18 @@ class AvatarController extends AbstractController
 {
     public function avatarCreate()
     {
-        // On crécupère le userId pour identification
-        $_SESSION ['userId'] = 3;
+        // On récupère le userId pour identification
+        //$_SESSION ['userId'] = 3;
```
Comment (aldaitzwild) [inline src/Controller/AvatarController.php:12]: No dead code !

File: src/View/Profiles/profile.html.twig [hunk 4]
```diff
@@ -1,32 +1,32 @@
...
-			</td>
-		</tr>
- 
-		<tr>
-			<td>
-				<label for ="Lastname">Lastname :</label>
-			</td>
-			<td text align="left">
-				<input type="text" placeholder="Prénom" value= "{{user.lastName}}" id="Lastname" name="Lastname"/>
+<div class="cadre">
```
Comment (aldaitzwild) [inline src/View/Profiles/profile.html.twig:4]: Warning div not closed

**Review by aldaitzwild — CHANGES_REQUESTED** [review]

**Review by aldaitzwild — CHANGES_REQUESTED** [review]
Conflicts with dev : 

Conflicting files
public/assets/css/style.css

**Review by aldaitzwild — CHANGES_REQUESTED** [review]
Still Dead code ...

**Review by aldaitzwild — CHANGES_REQUESTED** [review]
It's always the last zombie that kill everyone at the end : 

AvatarController.php => line 12  : Dead Code !

**Comment by Unikorn44** [comment]
No mOaR deAd cOd3 ?

**Review by aldaitzwild — APPROVED** [review]

**Pull request merged**